README.md

```diff
@@ -15,23 +15,23 @@
 
 1. Setup your application at [LinkedIn Developers](https://developer.linkedin.com/).
 
-1. Add `:ueberauth_linkedin` to your list of dependencies in `mix.exs`:
+2. Add `:ueberauth_linkedin` to your list of dependencies in `mix.exs`:
```

Comment:
The initial notation was correct - using `1.` everywhere automatically gets converted to numbered list and replaces with the correct numbers.

Reply:
I know, I just switched it up because I was reading the markdown file on its own.